core/templates/services/math-interactions.service.spec.ts

```diff
@@ -128,6 +128,24 @@ describe('MathInteractionsService', () => {
     expect(mathInteractionsService.getWarningText()).toBe(
       'Your answer has two symbols next to each other: "^" and "/".');
 
+    expect(mathInteractionsService.validateAlgebraicExpression(
+      'sqrt() + x', ['x'])).toBeFalse();
+    expect(mathInteractionsService.getWarningText()).toBe(
+      'The sqrt function(s) cannot be empty. ' +
+      'Please enter a variable/number in it.');
+
+    expect(mathInteractionsService.validateAlgebraicExpression(
+      'sin()/x', ['x'])).toBeFalse();
+    expect(mathInteractionsService.getWarningText()).toBe(
+      'The sin function(s) cannot be empty. ' +
+      'Please enter a variable/number in it.');
+
+    expect(mathInteractionsService.validateAlgebraicExpression(
+      'tan()sin()', ['x'])).toBeFalse();
+    expect(mathInteractionsService.getWarningText()).toBe(
+      'The sin,tan function(s) cannot be empty. ' +
```

Comment:
```suggestion
      'The sin, tan function(s) cannot be empty. ' +
```
May need to change the original code. Always have space after ,